code/game/objects/items/tcg/tcg.dm

```diff
@@ -0,0 +1,273 @@
+///A global list of cards, or rather changes to be applied to cards in the format
+
+GLOBAL_LIST_EMPTY_TYPED(card_list, /datum/card)
+
+/obj/item/tcgcard
+	name = "Coder"
+	desc = "Wow, a mint condition coder card! Better tell the Github all about this!"
+	icon = 'icons/obj/tcg.dmi'
+	icon_state = "runtime"
+	w_class = WEIGHT_CLASS_TINY
+	var/id = -1 //Unique ID, for use in lookups and storage, used to index the global datum list where the rest of the card's info is stored
+	var/flipped = 0
+
+/obj/item/tcgcard/Initialize(mapload, datum/card/temp)
+	. = ..()
+	name = temp.name
+	desc = temp.desc
+	icon = icon(temp.icon)
+	icon_state = temp.icon_state
+	id = temp.id
+	transform = matrix(0.3,0,0,0,0.3,0)
+
+/datum/card
+	var/id = -1 //Unique ID, for use in lookups and (eventually) for persistence. MAKE SURE THIS IS UNIQUE FOR EACH CARD, OR THE ENTIRE SYSTEM WILL BREAK, AND I WILL BE VERY DISAPPOINTED.
```

Comment:
This is super brittle code.

Reply:
The id? It's intended to allow for quick lookup and caching. It also means if/when this becomes persistent, we don't need to worry about legacy cards saved in the database, since all the info about what is what can be modified by the coders.

Reply:
Could you use the card name or type path as the ID? Relying on the coders to give it a unique number and failure of that causing issues is dangerous.